docs/getting-started-guides/windows/index.md

```diff
@@ -440,5 +440,5 @@ Some of these limitations will be addressed by the community in future releases
 - Horizontal Pod Autoscaling for Windows Server Container pods has not been verified to work end-to-end
 - Hyper-V Containers are not supported
 
-[!Warning]   
+![Warning]   
```

Comment:
This is a MSDN syntax; remove this line and either make the content a bullet point like the above issues or a block-quote by prefixing with `> `.

Reply:
updated